backend/FwLite/FwDataMiniLcmBridge/FwDataBridgeConfig.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace FwDataMiniLcmBridge;
+﻿using System.Runtime.InteropServices;
+
+namespace FwDataMiniLcmBridge;
 
 public class FwDataBridgeConfig
 {
-    public string ProjectsFolder { get; set; } = @"C:\ProgramData\SIL\FieldWorks\Projects";
-    public string TemplatesFolder { get; set; } = @"C:\Program Files\SIL\FieldWorks 9\Templates";
+    private static string UnixDataFolder =>
+        Environment.GetEnvironmentVariable("XDG_DATA_HOME") ??
+        Path.Join(Environment.GetEnvironmentVariable("HOME") ?? "", ".local", "share");
+
+    private static readonly string DataFolder =
+        RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+            ? @"C:\ProgramData\SIL\FieldWorks"
+            : Path.Join(UnixDataFolder, "fieldworks");
+    public string ProjectsFolder { get; set; } = Path.Join(DataFolder, "Projects");
+    public string TemplatesFolder { get; set; } = Path.Join(DataFolder, "Templates");
```

Comment:
Note that it's possible, on Linux at least, for the Templates folder not to exist. There's code that creates ProjectsFolder if it doesn't exist, but something similar needs to be done for the TemplatesFolder as well. On my system I had every test fail until I ran `mkdir -p /home/rmunn/.local/share/fieldworks/Templates/`.